components/companycam/actions/add-project/add-project.mjs

```diff
@@ -0,0 +1,123 @@
+import app from "../../companycam.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "companycam-add-project",
+  name: "Add Project",
+  description: "Add a new project. [See the docs](https://docs.companycam.com/reference/createproject).",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    app,
+    projectName: {
+      propDefinition: [
+        app,
+        "projectName",
+      ],
+    },
+    streetAddress1: {
+      type: "string",
+      label: "Street Address 1",
+      description: "The first line of the street address.",
+    },
```

Comment:
It seems a little weird to me to have `streetAddress1`, but not the rest of the address. Do you think it would make sense to add city, state, postal_code, & country as optional props?